Library/.rubocop_cask.yml

```diff
@@ -3,6 +3,8 @@ inherit_from: ./.rubocop_shared.yml
 Cask/HomepageMatchesUrl:
   Description: 'Ensure that the homepage and url match, otherwise add a comment. More info at https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment'
   Enabled: true
+  Exclude:
+  - '**/test/support/fixtures/cask/Casks/**/*.rb'
```

Comment:
The `Cask/HomepageMatchesUrl` cop was failing with any test cask that included a url with `”file://...“`. To not make this pull request too big, I thought it would make sense to exclude this check for now in the test cask directory and follow up with a fix to enable this cop after. What do you think?